### ISSUE TITLE
Support for PICO 2

### ISSUE THREAD

**Issue by SmakyTaky** [webhdx/PicoBoot#123]
| Answer some general questions first                             	|  	|
|:-------------------------------------------------------------------	|:------:	|
| Are you my GitHub sponsor? | no 	|
| Did you check older issues to see if it was requested before? | yes |

Will Raspberry Pico 2 be supported? I tried to complile the source code with both Visual Studio Core + Pico Extension and from command line (after installing cmake and phyton 3. Both would not complete complilation and show errors. Albeit I am a newbie when it comes to PICO compiling, I was not able to figure it out.

**Comment by thericeking** [comment]
Can we expect Pico 2 support sometime in 2025?
PS. Thank you for your work and Merry Christmas.

**Comment by webhdx** [comment]
Yes I already have the firmware ready, just need to test it. I hope to release it around Christmas.

**Comment by SmakyTaky** [comment]
These are great news!On Dec 21, 2024 10:16 AM, Maciej Kobus ***@***.***> wrote:
Yes I already have the firmware ready, just need to test it. I hope to release it around Christmas.

**Comment by VersatileNinja** [comment]
Checking to see if any plans to release pico 2 support this year?

**Comment by webhdx** [comment]
Yes I plan to work on it shortly.

**Comment by VersatileNinja** [comment]
Checking in, no rush.

**Comment by webhdx** [comment]
Check the latest release [v0.5.0](https://github.com/webhdx/PicoBoot/releases/tag/v0.5.0). Pico 2 support was added.
